techniques/system/common/1.0/promises.st

```diff
@@ -547,22 +548,6 @@ bundle agent check_log_system
     pass2.rsyslogd::
       "any" usebundle => rudder_common_report("Common", "log_info", "&TRACKINGKEY&", "Log system for reports", "None", "Detected running syslog as rsyslog");
 
-    pass2.((SuSE.(syslog_ng_repaired|rsyslog_repaired|syslogd_repaired))|((!SuSE.!aix.!solaris).syslogd_repaired))::
-
-      "restart_syslog"    usebundle => service_restart("syslog");
-
-    pass2.!SuSE.syslog_ng_repaired::
-
-      "restart_syslog_ng" usebundle => service_restart("syslog-ng");
-
-    pass2.!SuSE.rsyslog_repaired::
-
-      "restart_syslog_ng" usebundle => service_restart("rsyslog");
-
-    pass2.!SuSE.(syslog_ng_repaired|rsyslog_repaired|syslogd_repaired)::
-
-      "restart_syslog" usebundle => service_restart("syslog");
```

Comment:
this made no sense - on a not suse, if rsyslog were repaired, it would restart rsyslog and syslog !